xml/Microsoft.Extensions.Logging.Abstractions/NullLogger.xml

```diff
@@ -119,8 +119,9 @@
       </Parameters>
       <Docs>
         <param name="logLevel">To be added.</param>
```

Comment:
Can you document these params that did not get ported? Here and elsewhere below.

Reply:
I'll leave that for a future iteration if OK. now I'd want to focus on adding pre-existing docs.